package.json

```diff
@@ -59,8 +59,8 @@
         "@mdi/font": "^7.2.96",
         "@types/webfontloader": "^1.6.35",
         "@vue/test-utils": "^2.3.2",
-        "@vueuse/components": "^10.1.0",
-        "@vueuse/core": "^10.1.0",
+        "@vueuse/components": "^9.13.0",
+        "@vueuse/core": "^9.13.0",
```

Comment:
so vueuse is responsible for tesseract not working?

Reply:
so why reverting vueuse?

Reply:
Tought because of this

<img width="682" alt="image" src="https://user-images.githubusercontent.com/70775490/234072957-6baba10d-f688-48dc-b8e6-96772fa20e13.png">

Reply:
> Tought because of this
> 
> <img alt="image" width="682" src="https://user-images.githubusercontent.com/70775490/234072957-6baba10d-f688-48dc-b8e6-96772fa20e13.png">

there I was only talking about tesseract. I thought that error you posted had to do with a tesseract update

Reply:
Ahh ok no, the error only occours when I update the tesseract package to the latest version